tests/gtest/avifallocationtest.cc

```diff
@@ -188,13 +188,26 @@ TEST(EncodingTest, MaximumInvalidDimensions) {
                AVIF_RESULT_UNSUPPORTED_DEPTH);
 }
 
+#if defined(__clang__) && defined(__has_feature)
+// has to be on a different line for Windows.
```

Comment:
I think this is true in general if __has_feature isn't defined since the preprocessor doesn't short circuit.

Reply:
removed comment